### DISCUSSION TITLE
create-umi build 命令执行失败

### DISCUSSION THREAD

**Discussion by xiaodel** [umijs/umi#11712] [Q&A]
github create-umi 项目 build 命令执行失败
 "scripts": {
    "build": "umi-scripts father build",
    "build:deps": "umi-scripts bundleDeps",
    "dev": "umi-scripts father dev",
    "test": "umi-scripts jest-turbo"
  }
  
  PS E:\code\react\umi\packages\create-umi> npm run build

> create-umi@4.0.0-canary.20231007.1 build
> umi-scripts father build

'umi-scripts' 不是内部或外部命令，也不是可运行的程序
或批处理文件。

**Comment by fz6m** [comment]
如果你要开发 umi 源码仓库的话，要先在代码根目录执行 `pnpm i && pnpm build`
